### DISCUSSION TITLE
Hubbard model: SpinUnresolvedONVBasis

### DISCUSSION THREAD

**Discussion by dariavh** [GQCG/GQCP#884] [Ideas]
Currently the code offers an algorithm to calculate the Hubbard parameters for a `SpinResolvedONVBasis`. Would it be beneficially to extend the Hubbard code to `SpinUnresolvedBasis` objects as well? In @guacke's doctoral thesis, he optimized the Hubbard wavefunction for both spin-unresolved and spin-resolved electrons and subsequently investigated their MPDs.

**Comment by lelemmen** [comment]
> Would it be beneficially to extend the Hubbard code to `SpinUnresolvedBasis` objects as well? In @guacke's doctoral thesis, he optimized the Hubbard wavefunction for both spin-unresolved and spin-resolved electrons and subsequently investigated their MPDs.

Could you shed some light on how the Hubbard Hamiltonian is defined in a general spinor basis?
